examples/rbfe_single.py

```diff
@@ -0,0 +1,160 @@
+# This script computes the relative binding free energy of a single edge.
+
+import os
+import argparse
+import numpy as np
+
+from rdkit import Chem
+from rdkit.Chem import rdFMCS
+from rdkit.Chem import AllChem
+
+from fe import topology
+from md import builders
+from md import minimizer
+
+import functools
+
+from ff import Forcefield
+from ff.handlers.deserialize import deserialize_handlers
+
+import multiprocessing
+
+from fe import free_energy
+
+
+def wrap_method(args, fn):
+    gpu_idx = args[0]
+    os.environ['CUDA_VISIBLE_DEVICES'] = str(gpu_idx)
+    return fn(*args[1:])
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(
+        description="Relative Hydration Free Energy Consistency Testing",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--num_gpus",
+        type=int,
+        help="number of gpus",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_complex_windows",
+        type=int,
+        help="number of vacuum lambda windows",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_solvent_windows",
+        type=int,
+        help="number of solvent lambda windows",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_equil_steps",
+        type=int,
+        help="number of equilibration lambda windows",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_prod_steps",
+        type=int,
+        help="number of production lambda windows",
+        required=True
+    )
+
+
+    cmd_args = parser.parse_args()
+
+    multiprocessing.set_start_method('spawn') # CUDA runtime is not forkable
+    pool = multiprocessing.Pool(cmd_args.num_gpus)
+
+    suppl = Chem.SDMolSupplier('tests/data/ligands_40.sdf', removeHs=False)
+    all_mols = [x for x in suppl]
+    mol_a = all_mols[1]
+    mol_b = all_mols[4]
+
+    core = np.array([[ 0,  0],
+       [ 2,  2],
+       [ 1,  1],
+       [ 6,  6],
+       [ 5,  5],
+       [ 4,  4],
+       [ 3,  3],
+       [15, 16],
+       [16, 17],
+       [17, 18],
+       [18, 19],
+       [19, 20],
+       [20, 21],
+       [32, 30],
+       [26, 25],
+       [27, 26],
+       [ 7,  7],
+       [ 8,  8],
+       [ 9,  9],
+       [10, 10],
+       [29, 11],
+       [11, 12],
+       [12, 13],
+       [14, 15],
+       [31, 29],
+       [13, 14],
+       [23, 24],
+       [30, 28],
+       [28, 27],
+       [21, 22]]
+    )
+    ff_handlers = deserialize_handlers(open('ff/params/smirnoff_1_1_0_ccc.py').read())
+    ff = Forcefield(ff_handlers)
+
+    # the water system first.
```

Comment:
```suggestion
    # hif2a complex system
```

should we also mention here the source of this pdb file -- is it modified from https://github.com/MCompChem/fep-benchmark/tree/master/hif2a ?

Reply:
It's modified slightly with minimization in maestro and crystallographic waters fully stripped.